sample/src/androidTest/java/com/schibsted/spain/barista/sample/introduction/IntroducingBaristaActions.java

```diff
@@ -85,7 +90,7 @@ public void letsIntroduceBarista() {
     openDrawer(R.id.drawer);
     closeDrawer(R.id.drawer);
 
-    //Interact with SwipeRefreshLayout (pullToRefresh)
+    // Pull to refresh a SwipeRefreshLayout
```

Comment:
Ah! Didn't see you committing around xD Don't have to rollback, it's ok.
I'm saying it because it's not a small PR already, I think it's a good thing to keep it as clean as possible. No hurt feelings ❤️